shellbase-core/src/main/scala/com/sumologic/shellbase/ShellBase.scala

```diff
@@ -89,6 +89,11 @@ abstract class ShellBase(val name: String) {
   def historyPath: File = new File("%s/.%s_history".format(System.getProperty("user.home"), name))
 
   /**
+    * Hide in help built-in commands.
+    */
+  def hideInHelpBuiltInCommands(name: String): Boolean = false
```

Comment:
Also, this will require us to bump to 1.1.  (Anyone who uses this method + `override` will introduce a backwards incompatible change, so it feels like the version needs to be bumped.)